nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-processors-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,5 @@
+nifi-snowflake-processors-nar
+Copyright 2015-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
Is this one empty on purpose?

Reply:
I'm still not 100% sure about how to properly handle NOTICE files. This nar doesn't have any new dependency with `compile` scope. AFAIK in this case an empty NOTICE file should be provided. Could someone clarify this?

Reply:
AFAIK both empty or no notice files are fine in this case. I would leave it as it is now.